### DISCUSSION TITLE
Issue with 9.12 when loading into the Map10

### DISCUSSION THREAD

**Discussion by rneubauer** [carla-simulator/carla#4497] [General]
Congrats on the 912 Release, BTW!  I just downloaded and am running on my High end machine with an RTX3090. I have a separate linux machine that is running the server application. When I run the manual_control.py script form my machine, it connects fine, but then I get this in the view: 
![image](https://user-images.githubusercontent.com/1253132/128413377-3e8556a7-4a8d-4aaf-b529-41d963e92d5d.png)

Any idea what is causing it?

**Comment by wendytwh** [comment]
hi @rneubauer I’m also observing same over exposure problem in version 0.9.12,
I wonder if you have found any way around ?

**Comment by rneubauer** [comment]
For me, it turned out to be that I was using the built in driver fro Ubuntu, and not the nvidia driver. Once I switched it. it all started working fine. I was also having some performance problems, but that also got resolved as son as I switched to the right driver. Go Figure, LOL :P
